Lib/ldap/schema/subentry.py

```diff
@@ -464,6 +464,8 @@ def urlfetch(uri,trace_level=0,bytes_mode=None):
     if subschemasubentry_dn is None:
       s_temp = None
     else:
+      if isinstance(subschemasubentry_dn, bytes):
+        subschemasubentry_dn = subschemasubentry_dn.decode('utf-8')
```

Comment:
Hmm, I think you just found a nasty bug.

According to [RFC4512](https://tools.ietf.org/html/rfc4512#section-4.2), a subschemaSubentry is always a DN-type entry; which (by specification) *must* be UTF-8 encoded.

This means that this decoding bit should actually happen around the return value of [``search_subschemasubentry_s``](https://github.com/pyldap/pyldap/blob/master/Lib/ldap/ldapobject.py#L864).

Would you mind moving this part there? We should be all good from there :)